packages/wrangler/src/cloudchamber/images/registries.ts

```diff
@@ -42,23 +42,6 @@ function configureImageRegistryOptionalYargs(yargs: CommonYargsArgv) {
 		});
 }
 
-function credentialsImageRegistryYargs(yargs: CommonYargsArgv) {
-	return yargs
-		.positional("domain", { type: "string", demandOption: true })
-		.option("expiration-minutes", {
-			type: "number",
-			default: 15,
-		})
-		.option("push", {
-			type: "boolean",
-			description: "If you want these credentials to be able to push",
-		})
-		.option("pull", {
-			type: "boolean",
-			description: "If you want these credentials to be able to pull",
-		});
-}
-
```

Comment:
Is this intentional?

Reply:
Yes - these functions are not actually used at runtime, they were only created to make a type.
I could have put eslint exclusions around them but rather chose to get the types elsewhere.